lms/templates/student_account/account_settings.underscore

```diff
@@ -2,6 +2,7 @@
 <div class="account-settings-container">
     <div class="wrapper-header">
         <h2 class="header-title"><%- gettext("Account Settings") %></h2>
+        
```

Comment:
If the whitespace change is the only change in a file - you should not do it

Reply:
Fixed